### ISSUE TITLE
Moving default document backup directory [Question]

### ISSUE THREAD

**Issue by Rezonono** [OpenBoard-org/OpenBoard#883]
Hello,

I've just changed my computer and installed the Windows 1.7.0 version of OpenBoard on my C system disk (Windows 11).
As I had done on my old computer, for practical reasons I want to move the default document backup directory to D:/SAVE/OpenBoard.
I went through the same steps as on my old PC:
I opened the OpenBoard.config file in the .../OpenBoard/etc directory and added the following line:
DataDirectory=D:/SAVE/OpenBoard

It worked fine on my old PC with an older version of OpenBoard (1.5.4) but on my new PC with version 1.7.0, it doesn't work. It creates and saves my documents in an AVEpenBoard directory located in the software installation directory :
.../OpenBoard/AVEpenBoard

I've never heard of this directory and I haven't found anything about it.

I'd like to be able to move my document backups (which take up a lot of space) to a new location (D:/SAVE/OpenBoard).
Have I missed something?
If anyone has a solution, it would be a great help!

Thanks

**Comment by letsfindaway** [comment]
This very much looks as if you're using single backslashes like `D:\SAVE\OpenBoard` for the path. `\S` and `\O` aren't understood by Qt, so it makes `D:AVEpenBoard` out of it. To make it work as intended you have to use

- either double backslashes like `D:\\SAVE\\OpenBoard`
- or single forward slashes like `D:/SAVE/OpenBoard`.

Instead of using the `OpenBoard.config` file, which is overwritten with each installation of a new version of OpenBoard, you could also use the `OpenBoardUser.config` file as explained in [Configuration](https://github.com/OpenBoard-org/OpenBoard/wiki/Configuration). On Windows, this file is located in your user profile at `%USERPROFILE%\AppData\Local\OpenBoard\`. Every setting in this file overrules the default from `OpenBoard.config`. Also any configuration changes you make using the Preferences dialog are saved in this file.

**Comment by Rezonono** [comment]
Many thanks for your help.

For information I did not succeed using the file OpenBoardUser.config located in my user profile at %USERPROFILE%\AppData\Local\OpenBoard\
It didn't override the file from OpenBoard.config
So I edited the OpenBoard.config file and added the line :
DataDirectory=D:\\SAVE\\OpenBoard
(D:/SAVE/OpenBoard did not work).

I'll keep your post in mind.
Thank you very much!

**Comment by letsfindaway** [comment]
> For information I did not succeed using the file `OpenBoardUser.config` located in my user profile at `%USERPROFILE%\AppData\Local\OpenBoard\` It didn't override the file from `OpenBoard.config` So I edited the `OpenBoard.config file` and added the line : `DataDirectory=D:\\SAVE\\OpenBoard` (`D:/SAVE/OpenBoard` did not work).

You are right with the first part: The `DataDirectory` setting must be in `OpenBoard.config`. As it also determines the storage location of the user configuration file, it cannot logically be set there.

But single forward slashes should definitely work, too.
